scripts/ci/check_compliance.py

```diff
@@ -1221,8 +1221,8 @@ def _main(args):
     return n_fails
 
 
-def main():
-    args = parse_args()
+def main(argv):
```

Comment:
We can make it optional by doing `def main(argv=None):` (which is a typical style for Python main functions)

Reply:
Can't argue with Python traditions. Done.